app/src/main/java/de/westnordost/streetcomplete/quests/note_discussion/NoteDiscussionForm.kt

```diff
@@ -140,22 +141,20 @@ class NoteDiscussionForm : AbstractQuestAnswerFragment<NoteAnswer>() {
             val userName = if (comment.user != null) comment.user.displayName else getString(R.string.quest_noteDiscussion_anonymous)
 
             val commentActionResourceId = comment.action.actionResourceId
-            if (commentActionResourceId != 0) {
-                itemView.commentStatusText.visibility = View.VISIBLE
+            val isStatusGone = commentActionResourceId == 0
+            itemView.commentStatusText.isGone = isStatusGone
+            if (!isStatusGone) {
                 itemView.commentStatusText.text = getString(commentActionResourceId, userName, dateDescription)
-            } else {
-                itemView.commentStatusText.visibility = View.GONE
             }
 
-            if (!comment.text.isNullOrEmpty()) {
-                itemView.commentView.visibility = View.VISIBLE
+            val isCommentGone = comment.text.isNullOrEmpty()
```

Comment:
actually, it means that there is no comment. For example, if you just resolve an OSM note without providing an additional comment. So I suggest to rename it !`hasComment`